command/cp.go

```diff
@@ -70,6 +70,9 @@ Examples:
 
 	12. Perform KMS-SSE of the object(s) at the destination using customer managed Customer Master Key (CMK) key id
 		> s5cmd {{.HelpName}} -sse aws:kms -sse-kms-key-id <your-kms-key-id> s3://bucket/object s3://target-bucket/prefix/object
+
+	13. Check what s5cmd will do, without actually doing so
+		> s5cmd {{.HelpName}} --dry-run dir/ s3://bucket/ 
```

Comment:
In `dry-run` mode `cp` command prepares local destination which introduces side effects.

For example below command creates directory `~/TEMP/ilkin` if it does not exists.
```shell
s5cmd cp --dry-run  's3://test-bucket/files*'  ~/TEMP/ilkin/
```

Reply:
thanks for pointing it out

Reply:
Can you add tests for upload and download operations? As far as I see, all dry-run tests are covering s3 copy (s3->s3). We can add assertions to show whether directories/files created.

Reply:
@igungor yeah, it's addressed. thanks for the tip, will do as such from now on.